api/src/test/scala/hmda/api/http/institutions/submissions/SubmissionEditPathsSpec.scala

```diff
@@ -211,27 +217,34 @@ class SubmissionEditPathsSpec extends InstitutionHttpApiSpec with LarGenerators
   }
 
   "Edit Type endpoint: return 405 when posting justification to syntactical endpoint" in {
-    postWithCfpbHeaders("/institutions/0/filings/2017/submissions/0/edits/syntactical") ~> institutionsRoutes ~> check {
+    postWithCfpbHeaders("/institutions/0/filings/2017/submissions/0/edits/syntactical") ~> Route.seal(institutionsRoutes) ~> check {
```

Comment:
Since I changed how `SubmissionEditPaths` responds to the "Justify Macro Edits" endpoint, we no longer have to manually return a "Method Not Allowed" error (that's why I could remove the `completeWithMethodNotAllowed` method in `HmdaCustomDirectives`). Now, akka-http handles that rejection automatically, but only when the route is "sealed".